src/modules/ManagedItems.js

```diff
@@ -108,7 +108,10 @@ export class ManagedItems {
 	get propValues() {
 		const values = {}
 		Object.keys(this.propDefs).forEach((propName) => {
-			values[propName] = this.getPropValues(propName)
+			let valueFn = this.propDefs[propName].valueFn
+				? this.propDefs[propName].valueFn
+				: (node) => node[propName]
+			values[propName] = this.getPropValues(propName, valueFn)
```

Comment:
Cannot you just write:

```suggestion
			values[propName] = this.getPropValues(propName, this.propDefs[propName].valueFn)
```